### PR TITLE
Translations update from Hosted Weblate

### PR DESCRIPTION
**Pull request by weblate**
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [Jamulus/Startpage](https://hosted.weblate.org/projects/jamulus/1-index/).

It also includes the following components:

* [Jamulus/Command-Line-Options](https://hosted.weblate.org/projects/jamulus/command-line-options/)

* [Jamulus/Copyright notices](https://hosted.weblate.org/projects/jamulus/copyright-notices/)

* [Jamulus/Software-Manual](https://hosted.weblate.org/projects/jamulus/software-manual/)

* [Jamulus/Include-Shared-Commands](https://hosted.weblate.org/projects/jamulus/include-shared-commands/)

* [Jamulus/Include-Backing-Up](https://hosted.weblate.org/projects/jamulus/include-backing-up/)

* [Jamulus/General strings](https://hosted.weblate.org/projects/jamulus/general/)

* [Jamulus/Unregistered-Servers](https://hosted.weblate.org/projects/jamulus/unregistered-servers/)

* [Jamulus/Include-Client-Commands](https://hosted.weblate.org/projects/jamulus/include-client-commands/)

* [Jamulus/Server-Troubleshooting](https://hosted.weblate.org/projects/jamulus/server-troubleshooting/)

* [Jamulus/QOS-Windows](https://hosted.weblate.org/projects/jamulus/qos-windows/)

* [Jamulus/Server-Bandwidth](https://hosted.weblate.org/projects/jamulus/server-bandwidth/)

* [Jamulus/Contribution](https://hosted.weblate.org/projects/jamulus/contribution/)

* [Jamulus/Installation-for-Macintosh](https://hosted.weblate.org/projects/jamulus/installation-for-macintosh/)

* [Jamulus/Getting-Started](https://hosted.weblate.org/projects/jamulus/getting-started/)

* [Jamulus/Installation-for-Android](https://hosted.weblate.org/projects/jamulus/installation-for-android/)

* [Jamulus/Installation-for-iOS](https://hosted.weblate.org/projects/jamulus/installation-for-ios/)

* [Jamulus/Client-Troubleshooting](https://hosted.weblate.org/projects/jamulus/client-troubleshooting/)

* [Jamulus/FAQ](https://hosted.weblate.org/projects/jamulus/faq/)

* [Jamulus/Tips-Tricks-More](https://hosted.weblate.org/projects/jamulus/tips-tricks-more/)

* [Jamulus/Navigation](https://hosted.weblate.org/projects/jamulus/navigation/)

* [Jamulus/Installation-for-Linux](https://hosted.weblate.org/projects/jamulus/installation-for-linux/)

* [Jamulus/Installation-for-Windows](https://hosted.weblate.org/projects/jamulus/installation-for-windows/)

* [Jamulus/Running-a-Server](https://hosted.weblate.org/projects/jamulus/running-a-server/)

* [Jamulus/Directories](https://hosted.weblate.org/projects/jamulus/directories/)

* [Jamulus/Privacy-Statement](https://hosted.weblate.org/projects/jamulus/privacy-statement/)

Current translation status:

Weblate translation status

### CODE REVIEW

File: _translator-files/po/de/Getting-Started.po [hunk 1]
```diff
@@ -56,24 +55,18 @@ msgstr "Damit du Jamulus gut nutzen kannst, solltest du mindestens das Folgende
...
-msgstr "**Kabelgebundene Kopfhörer (kein Bluetooth Kopfhörer oder Lautsprecher** - siehe [FAQ](/wiki/FAQ#warum-sollte-ich-keine-drahtlosen-geräte-verwenden))\n"
+msgstr "**Kabelgebundene Kopfhörer (kein Bluetooth oder Lautsprecher** - siehe die [FAQ](/wiki/FAQ#warum-sollte-ich-keine-drahtlosen-geräte-verwenden))"
 
 #. type: Bullet: '1. '
 #: ../wiki/en/Getting-Started.md:15
-#, fuzzy
-#| msgid "**A reasonable audio device, sound card and/or mic** ([see this list](/kb/2021/01/05/Jamulus-Sound-Devices.html) for examples)\n"
 msgid "**A reasonable audio device, sound card and/or mic** ([see this list](/kb/2021/01/05/Jamulus-Sound-Devices.html) for examples)"
-msgstr "**Ein passendes Audiointerface oder Soundkarte und Mikrofon** ([siehe diese Auflistung](/kb/2021/01/05/Jamulus-Sound-Devices.html) für Beispiele)\n"
+msgstr "**Eine passende Audioschnittstelle, Soundkarte und/oder Mikrofon** ([siehe diese Auflistung](/kb/2021/01/05/Jamulus-Sound-Devices.html) für Beispiele)"
```
Comment (ann0see) [inline _translator-files/po/de/Getting-Started.po]:
```suggestion
msgstr "**Ein passendes Audiointerface/Soundkarte und/oder Mikrofon** ([siehe diese Auflistung](/kb/2021/01/05/Jamulus-Sound-Devices.html) für Beispiele)"
```

**Review by ann0see — COMMENTED** [review]
@jujudusud @ignotus666 @trebmuh please cross review this PR.

File: _translator-files/po/de/Getting-Started.po (hunk 1 above)
Comment (trebmuh) [reply _translator-files/po/de/Getting-Started.po]: @ann0see all good from my point of view

Comment (ann0see) [reply _translator-files/po/de/Getting-Started.po]: Thanks!

**Review by jujudusud — APPROVED** [review]
French part is ok from my point of view.
Thanks to @trebmuh.

**Comment by ann0see** [comment]
Waiting on review by Ignotus, then feel free to merge this. 
The German suggestion isn't that critical but would be good to have.

File: _translator-files/po/de/Getting-Started.po (hunk 1 above)
Comment (jujudusud) [reply _translator-files/po/de/Getting-Started.po]: Do we commit this suggestion ?

**Review by ignotus666 — APPROVED** [review]

File: _translator-files/po/de/Getting-Started.po (hunk 1 above)
Comment (ann0see) [reply _translator-files/po/de/Getting-Started.po]: I changed it on Weblate

**Pull request merged**